stdlib/2and3/zipfile.pyi

```diff
@@ -22,28 +22,28 @@ class ZipFile:
     debug = ...  # type: int
     comment = ...  # type: bytes
     filelist = ...  # type: List[ZipInfo]
-    def __init__(self, file: Union[str, IO[bytes]], mode: str = ..., compression: int = ...,
+    def __init__(self, file: Union[Text, IO[bytes]], mode: Text = ..., compression: int = ...,
                  allowZip64: bool = ...) -> None: ...
     def __enter__(self) -> ZipFile: ...
     def __exit__(self, exc_type: Optional[Type[BaseException]],
                  exc_val: Optional[Exception],
                  exc_tb: Optional[TracebackType]) -> bool: ...
     def close(self) -> None: ...
-    def getinfo(self, name: str) -> ZipInfo: ...
+    def getinfo(self, name: Text) -> ZipInfo: ...
     def infolist(self) -> List[ZipInfo]: ...
-    def namelist(self) -> List[str]: ...
-    def open(self, name: _SZI, mode: str = ...,
+    def namelist(self) -> List[Text]: ...
+    def open(self, name: _SZI, mode: Text = ...,
              pwd: Optional[bytes] = ...) -> IO[bytes]: ...
     def extract(self, member: _SZI, path: Optional[_SZI] = ...,
                 pwd: bytes = ...) -> str: ...
-    def extractall(self, path: Optional[str] = ...,
-                   members: Optional[List[str]] = ...,
+    def extractall(self, path: Optional[Text] = ...,
+                   members: Optional[List[Text]] = ...,
```

Comment:
This would no longer accept a `List[str]`. From the implementation, it looks like it doesn't require strictly a list; can you change it to `Optional[Iterable[Text]]`?

Reply:
(Made this change myself.)